src/methods/createAndSendTx.ts

```diff
@@ -50,7 +50,8 @@ export async function createAndSendTx(
       });
     },
   };
-  await txExtrinsic.signAndSend(address, { signer });
+  let options=txArgs.immortality?{ signer,era:0 }:{ signer }
```

Comment:
We should probably start to enforce formatting on that repo

Reply:
done